app/assets/javascripts/maps.coffee.erb

```diff
@@ -107,6 +107,11 @@ KS.initializeMaps = ->
       KS.Map.updateSize()
     ), 500
 
+    setTimeout (->
+      KS.Map.getView().fit KS.findLayerById('features').getSource().getExtent(), KS.Map.getSize()
+      KS.Map.getView().setZoom(KS.Map.getView().getZoom() - 0.15)
+    ), 500
+
```

Comment:
Überlagern / behindern sich die beiden Funktionsblöcke nicht? 
Ist der `updateSize()` vorher noch notwendig? Falls ja dann wäre es doch besser beides nacheinander in das gleiche `setTimeout()` zu übernehmen, oder?

Reply:
Timeouts sind zusammengefügt. Ganz wegnehmen würde ich es nicht um irgendwelche Seiteneffekte auszuschließen